cmd/manager/main.go

```diff
@@ -47,34 +47,37 @@ func init() {
 
 func main() {
 	var (
-		metricsAddr                   string
-		enableLeaderElection          bool
-		probeAddr                     string
-		secureMetrics                 bool
-		enableHTTP2                   bool
-		macPrefixesFile               string
-		insecure                      bool
-		managerNamespace              string
-		probeImage                    string
-		probeOSImage                  string
-		registryPort                  int
-		registryProtocol              string
-		registryURL                   string
-		registryResyncInterval        time.Duration
-		webhookPort                   int
-		enforceFirstBoot              bool
-		enforcePowerOff               bool
-		serverResyncInterval          time.Duration
-		powerPollingInterval          time.Duration
-		powerPollingTimeout           time.Duration
-		resourcePollingInterval       time.Duration
-		resourcePollingTimeout        time.Duration
-		discoveryTimeout              time.Duration
-		serverMaxConcurrentReconciles int
+		metricsAddr                        string
+		enableLeaderElection               bool
+		probeAddr                          string
+		secureMetrics                      bool
+		enableHTTP2                        bool
+		macPrefixesFile                    string
+		insecure                           bool
+		managerNamespace                   string
+		probeImage                         string
+		probeOSImage                       string
+		registryPort                       int
+		registryProtocol                   string
+		registryURL                        string
+		registryResyncInterval             time.Duration
+		webhookPort                        int
+		enforceFirstBoot                   bool
+		enforcePowerOff                    bool
+		serverResyncInterval               time.Duration
+		powerPollingInterval               time.Duration
+		powerPollingTimeout                time.Duration
+		resourcePollingInterval            time.Duration
+		resourcePollingTimeout             time.Duration
+		discoveryTimeout                   time.Duration
+		serverMaxConcurrentReconciles      int
+		serverClaimMaxConcurrentReconciles int
 	)
 
 	flag.IntVar(&serverMaxConcurrentReconciles, "server-max-concurrent-reconciles", 5,
 		"The maximum number of concurrent Server reconciles.")
+	flag.IntVar(&serverClaimMaxConcurrentReconciles, "server-claim-max-concurrent-reconciles", 5,
```

Comment:
Maybe we should keep the default to one to be on the safe side and bump it up later? wdyt?